cmd/kube-apiserver/app/server.go

```diff
@@ -90,14 +89,14 @@ cluster's shared state through which all other components interact.`,
 	return cmd
 }
 
-// Run runs the specified APIServer.  This should never exit.
-func Run(s *options.ServerRunOptions) error {
+// Run runs the apiserver and configures it to stop with the given channel.
```

Comment:
Please add a sentence about the error case here, compare https://github.com/kubernetes/kubernetes/pull/42278/files#diff-ba08be166e7fd2d15ac0e7e725f7e2e7R79.